vchelaru/FlatRedBall
# 621 - Theming

Fixes: #621

This will start as a draft for user testing.

Testers: 
- Make sure you have latest version of Gum pulled and built. 
- Make sure to rebuild the entire solution as there have been changes made to plugin projects that aren't rebuilt when running glue (you should make habit to do this every time you pull any new changes)

**From the top menu strip, `Settings -> Theming` will bring up the theming window.**

I need help finding things areas that I haven't explored yet, as I'm not familiar with most features/aspects of the app and touched only the most obvious surfaces.

Please help me find:
- Lingering buttons which are still using icons generic enough to be replaced with [these material design icons](https://pictogrammers.com/library/mdi/)
- Areas which still appear to have hard coded colors that conflict with the theming changes.
- General visual fallout from the change, including: layout, padding, margin, size, unresponsive/unfit colors, etc.
- Any possibly functionality issues or visual inconsistencies specifically with DataGrids and ComboBoxes, as these controls have been re-templated, but the depth at which they need to be is currently unknown to me.

Use this PR as a place to communicate your findings, with screenshots and descriptions.

Thank you!
